Tools/c-analyzer/cpython/_generate.py

```diff
@@ -183,7 +183,6 @@ def find_matching_variable(varid, varcache, allfilenames, *,
     'float_format': 'static float_format_type ',
     'detected_double_format': 'static float_format_type ',
     'detected_float_format': 'static float_format_type ',
-    # Parser/listnode.c
     'level': 'static int ',
```

Comment:
These two need to go (level and atbol are the functions in listnode.c).

Reply:
> These two need to go (level and atbol are the functions in listnode.c, the same that you removed in ` Tools/c-analyzer/TODO`)

Whoops, didn't know that. Sorry!